backend/parser/examples/helloworld.java

```diff
@@ -9,4 +9,14 @@ public static void main(String[] args) {
    		}
 
    }
+
+   public static final void boo (){
+   		if(true) {
+   			return;
+   		}
+   }
+
+   public Bool foo(){
```

Comment:
Shouldn't use object based primitive types if you can use plain primitive types instead. Requires less memory and doesn't have to unwrapped. e.g Rather "bool" than "Bool"